src/style.css

```diff
@@ -0,0 +1,232 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  left: auto;
+  right: auto;
+
+  margin-top: 25px;
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+
+  max-width: 1200px;
+  width: 100%;
+  min-width: 1024px;
+  height: 60px;
+  padding-right: 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+}
+
+li {
+  padding: 22.5px 10px;
+  list-style: none;
+
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+
+  color: #000;
+}
+
+.is-active :hover {
+  color: #00acdc;
+}
+
+.item-1 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-2 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 58px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-3 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 156px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-4 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 163px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-5 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 59px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-6 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 70px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-7 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 33px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.item-8 :active::after {
+  color: #00acdc;
+
+  background-color: #00acdc;
+  content: "";
+  display: block;
+  width: 41px;
+  height: 4px;
+  border-radius: 8px;
+}
```

Comment:
Don't repeat yourself. You can move common properties to common class and use it

Reply:
Thank you! I have fixed and deployed it.